src/js/components/footer.js

```diff
@@ -46,11 +46,14 @@ export const Footer = () => {
   const { classes } = useStyles();
   return (
     <div className={classes.footer}>
-      {targets.map(({ key, target, title }) => (
-        <a className="clickable" href={target} key={key} target="_blank" rel="noopener noreferrer">
-          {title}
-        </a>
-      ))}
+      {targets.map(({ key, target, title }) => {
+        return (
```

Comment:
this return added on purpose, right? I am having doubts.

Reply:
Yes, I need it to add the comment (as this is not a single value anymore)

Reply:
ah right. sorry for the nosie.